avaframe/in1Data/getInput.py

```diff
@@ -516,69 +509,65 @@ def fetchReleaseFile(inputSimFiles, releaseScenario, cfgSim, releaseList):
         for scenario in releaseList:
             if scenario == releaseScenario:
-                cfgSim['INPUT']['relThId'] = cfgSim['INPUT'][scenario + '_' + 'relThId']
-                cfgSim['INPUT']['relThThickness'] = cfgSim['INPUT'][scenario + '_' + 'relThThickness']
-                cfgSim['INPUT']['relThCi95'] = cfgSim['INPUT'][scenario + '_' + 'relThCi95']
+                cfgSim["INPUT"]["relThId"] = cfgSim["INPUT"][scenario + "_" + "relThId"]
+                cfgSim["INPUT"]["relThThickness"] = cfgSim["INPUT"][scenario + "_" + "relThThickness"]
+                cfgSim["INPUT"]["relThCi95"] = cfgSim["INPUT"][scenario + "_" + "relThCi95"]
             # remove thickness, id and ci95 values specified by releaseScenario
-            cfgSim['INPUT'].pop(scenario + '_' + 'relThId')
-            cfgSim['INPUT'].pop(scenario + '_' + 'relThThickness')
-            cfgSim['INPUT'].pop(scenario + '_' + 'relThCi95')
+            cfgSim["INPUT"].pop(scenario + "_" + "relThId")
+            cfgSim["INPUT"].pop(scenario + "_" + "relThThickness")
+            cfgSim["INPUT"].pop(scenario + "_" + "relThCi95")
 
     return releaseScenarioPath, cfgSim
 
 
 def createReleaseStats(avaDir, cfg):
-    """ create a csv file with info on release shp file on:
-        max, mean and min elevation, slope and projected and real area
+    """create a csv file with info on release shp file on:
+    max, mean and min elevation, slope and projected and real area
 
-        Parameters
-        -------------
+    Parameters
+    -------------
 
-        Returns
-        --------
-        fPath: pathlib Path
-            file path
+    Returns
+    --------
+    fPath: pathlib Path
+        file path
     """
 
     # fetch input data (dem and release area shp file)
     inputData = getInputDataCom1DFA(avaDir)
 
     # get line from release area polygon
-    relFiles = inputData['relFiles']
+    relFiles = inputData["relFiles"]
 
     # initialize dem and get real area of dem
-    dem = IOf.readRaster(inputData['demFile'], noDataToNan=True)
-    dem['originalHeader'] = dem['header'].copy()
-    methodMeshNormal = cfg.getfloat('GENERAL', 'methodMeshNormal')
+    dem = IOf.readRaster(inputData["demFile"], noDataToNan=True)
+    dem["originalHeader"] = dem["header"].copy()
+    methodMeshNormal = cfg.getfloat("GENERAL", "methodMeshNormal")
     # get normal vector of the grid mesh
     dem = DFAtls.getNormalMesh(dem, methodMeshNormal)
     dem = DFAtls.getAreaMesh(dem, methodMeshNormal)
-    header = dem['header']
-    csz = header['cellsize']
 
     # loop over all relFiles and compute information saved to dictionary of dataframes
-    relInfo = {}
-    relPath = pathlib.Path(avaDir, 'Outputs', 'com1DFA', 'releaseInfoFiles')
+    relPath = pathlib.Path(avaDir, "Outputs", "com1DFA", "releaseInfoFiles")
     fU.makeADir(relPath)
     relDFDict = {}
     # loop over all relFiles
     for relFile in relFiles:
         # for relF in relFiles:
-        releaseLine = {}
```

Comment:
if it is not reinitialized as empty dict, doesnt it add/overwrite the keys?

Reply:
Mh, I'm pretty sure the keys are being overwritten, which is exactly what we want, right?